tutorials/oauth2-with-23andMe-in-java/index.md

```diff
@@ -1,3 +1,11 @@
+---
+title: 23andMe API & google oauth2 in java
```

Comment:
Some words here need to be capitalized as this is an actual page title. I suggest roughly something like:

    Authenticating to 23andMe API with Google OAuth client